### DISCUSSION TITLE
Loading the web debug toolbar failed

### DISCUSSION THREAD

**Discussion by ghost** [symfony/symfony#48308] [Q&A] [answered]
### Symfony version(s) affected

6.1

### Description

I've made a new project according to the docs (https://symfony.com/doc/current/setup.html):

> symfony new my_project_directory --version="6.1.*" --webapp

When I opened it in localhost or in local domain, the Symfony profiler doesn't work correctly. It just counts attemps to load the web debug toolbar. Debug mode is enabled as it always is right after creating new project. 

I will not copy here any code, because I didn't touch anything... 

### How to reproduce

.

**Comment by ghost** [comment]
this should be at least bad documention, not discussion. Thanks

**Comment by avtarsm** [comment, marked as answer]
Hi,

You will need to install symfony apache pack. 

composer require symfony/apache-pack

This should resolve the toolbar issue.

**Reply by ghost** [reply]
it does. thank you!
